accounting/gst.js

```diff
@@ -150,11 +189,79 @@ async function generateB2bData(invoices) {
 }
 
 async function generateB2clData(invoices) {
-  return [];
+  const b2cl = [];
+
+  invoices.forEach(async (invoice) => {
+
+    const stateInvoiceRecord = {
+      pos: posMap[invoice.place.toUpperCase()],
```

Comment:
I'm thinking if we should have a way to handle this better, because what if there's a mis-spelling and what if user enters the short form like JH or MH.

Is it possible to add a place of supply dropdown in regional changes when gst is enabled?

Reply:
@ankitsinghaniyaz that is what exactly I was thinking, ERPNext have this kind of dropdown and I am thinking to put that feature in here. Also we already have a SELECT (dropdown) component. This will remove the chances of error by many folds.

Reply:
Done 💯 
cc: @ankitsinghaniyaz